integration-tests/pom.xml

```diff
@@ -141,6 +141,8 @@
         <module>reactive-messaging-kafka</module>
         <module>reactive-messaging-http</module>
         <module>rest-client</module>
+        <module>rest-client-exception</module>
+        <module>rest-client-exception-original-behavior</module>
```

Comment:
Does it really make sense to add two new integration tests modules for that? Can't we test that in tests in the `deployment` module?

Reply:
@gsmet Sure, I can do it

Reply:
@gsmet done :-), let me know please if you'd like some more tweaks there